usagov/usagov-2021
# USAGOV-1422-report-issue-form-padding: update padding for mobile to c…

…enter the form


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1422

## Description

The form fields to stay centered when the screen width is less than 640px and not shift to the left. 
## Type of Changes

  - Add screenshot showing what it should look like
<img width="1512" alt="Screenshot 2024-03-12 at 11 02 06 AM" src="https://github.com/usagov/usagov-2021/assets/5551880/1f10d758-569a-4885-bfa6-88b151cf0ee3">

## Testing Instructions


Goto http://localhost/site-issue-report-form and inspect the form at a width just under 640 to verify the form stays centered and not shift to the left
### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
